team_delta.js

```diff
@@ -1 +1,26 @@
 // Fetch all routes for the airline and insert into #delta
+const getFlight = async (from, to, resourse) => {
+  const response = await fetch(resourse);
```

Comment:
Maybe construct the resource (also typo) from the `from` and `to` in a separate function, so if the API url changes it will be fixable only in one place?